crates/admin/src/cluster_controller/service.rs

```diff
@@ -248,26 +248,36 @@ impl<T: TransportConnect> Service<T> {
             )?;
         }
 
+        self.task_center.spawn_child(
+            TaskKind::SystemService,
+            "cluster-controller-metadata-sync",
+            None,
+            sync_cluster_controller_metadata(self.metadata.clone()),
+        )?;
+
+        let configuration = self.configuration.live_load();
+
         let mut scheduler = Scheduler::init(
+            configuration,
             self.task_center.clone(),
             self.metadata_store_client.clone(),
             self.networking.clone(),
         )
         .await?;
 
         let mut logs_controller = LogsController::init(
+            configuration,
```

Comment:
Not a blocker, but I'd have loved if we rely more on Live configuration for a more reactive experience.

Reply:
For what it's worth, the service use the `Live<Configuration>` and here we are only using a snapshot to setup the `LogsController`. Apart from this place, the `LogsController` does not need access to the `Configuration` so far.